com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -61,6 +61,7 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
         internal NetworkTickSystem NetworkTickSystem { get; private set; }
 
         internal SnapshotSystem SnapshotSystem { get; private set; }
+        internal NetworkBehaviourUpdater BehaviourUpdater { get; private set; }
```

Comment:
again, we're adding a brand-new feature to MLAPI/Runtime/Core and even plugging it into NetworkManager. I'm really curious what was the intention behind this.

Reply:
see answer above